src/Microsoft.AspNet.TestHost/HttpResponseMessageExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNet.TestHost
+{
+    public static class HttpResponseMessageExtensions
+    {
+        public static Task PipelineCompleteAsync(this HttpResponseMessage message)
```

Comment:
Why is the response stream completing before the end of the pipeline? Is it checking content-length? Or did somebody call Dispose on it?